front/src/modules/trainschedule/components/ImportTimetableItem/helpers/parseXML.ts

```diff
@@ -15,7 +15,7 @@ const mapTrainNames = (trainSchedules: TrainSchedule[], trains: Element[]): Trai
 
   trains.forEach((train) => {
     const trainPartRef = train.getElementsByTagName('trainPartRef')[0]?.getAttribute('ref');
-    const trainName = train.getAttribute('name') || '';
+    const trainName = train.getAttribute('description') || train.getAttribute('name') || '';
```

Comment:
We could maybe also add `trainNumber` as a fallback
```suggestion
    const trainName = train.getAttribute('description') || train.getAttribute('name')  || train.getAttribute('trainNumber') || '';
```